.github/workflows/basic.yml

```diff
@@ -14,8 +14,9 @@ jobs:
         command:
           - pytest
           - DIRAC_USE_M2CRYPTO=Yes pytest
-          - pytest Core/Security/test
-          - DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test
+          # Security tests are flakey due to reference counting bugs in pyGSI/src/crypto/asn1.c
+          - pytest Core/Security/test || (echo "Retrying..." pytest Core/Security/test) || (echo "Retrying again..." pytest Core/Security/test)
+          - DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test || (echo "Retrying..." DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test) || (echo "Retrying again..." DIRAC_USE_M2CRYPTO=Yes pytest Core/Security/test)
```

Comment:
are you sure about this?

Reply:
Yeah, whatever infrastructure quirk it is sensitive to isn't present at the moment but I expect the problem will randomly come back at some point. I don't see the harm in doing this until PyGSI is dropped entirely.

(Details in https://github.com/DIRACGrid/DIRAC/pull/4381#discussion_r366752264)

Reply:
Isn't this missing some ";" between the echo and the pytest command?